src/main/java/com/todo/todo/service/TodoService.java

```diff
@@ -0,0 +1,17 @@
+package com.todo.todo.service;
+
+import com.todo.todo.model.Todo;
+import java.util.List;
+
+public interface TodoService
```

Comment:
I see an interface here, but I don't see an implementation of any of these methods!